force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +430,23 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
-
+        Map<String, Integer> fruitQuantities = new Map<String, Integer>{
+        
         // Add key-value pairs to the Map
         // add Apples => 5
+        'Apples'=> 5,
         // add Oranges => 10
+        'Oranges' => 10,
         // add Bananas => 15
+        'Bananas' => 15,
         // add Pears => 20
+        'Pears' => 20,
         // add Grapes => 25
+        'Grapes' => 25
```

Comment:
Make sure you know how to use the `put()` method as well to add key/value pairs to a map. You'll use it a lot in Apex when dealing with object maps.